generated/nidcpower/tox-system_tests.ini

```diff
@@ -41,8 +41,8 @@ deps =
     nidcpower-system_tests: hightime
     nidcpower-system_tests: fasteners
     nidcpower-system_tests: pytest-json
-    nidcpower-system_tests: grpcio
-    nidcpower-system_tests: protobuf
+    nidcpower-system_tests: grpcio >= 1.49.1, < 2.0
+    nidcpower-system_tests: protobuf >= 4.21, < 5.0
```

Comment:
Does nimi-python run any tests without grpcio and protobuf installed?

Reply:
nimodinst, nise and nitclk all run system tests without grpcio and protobuf installed.

Reply:
Right, but nimodinst, nise, and nitclk don't support the "grpc" extra. If a package supports extra dependencies, it ought to have some test coverage without the extra dependencies installed to ensure that it still works without them. 

When I asked @mshafer-NI whether nidaqmx-python should declare grpcio and protobuf as extra dependencies, this seemed to be his number one concern with that approach.

Reply:
It sounds like you're concerned about a possible pre-existing test gap.
Come to think of it, we run our examples as part of our testing. Those examples don't import grpcio or protobut, but it's in the same venv as our system tests, so the extra dependencies **are** installed.